package.json

```diff
@@ -115,8 +115,9 @@
         "eslint": "8.52.0",
         "http-server": "14.1.1",
         "husky": "8.0.3",
-        "jest": "27.5.1",
-        "jest-preset-angular": "11.1.2",
+        "jest": "28.1.3",
+        "jest-environment-jsdom": "28.1.3",
```

Comment:
https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28/#jsdom